custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -200,7 +203,7 @@ def related_dates_changed(case):
         action for action in last_update_actions
```

Comment:
Would be good to at some point change these to the `case_properties_changed` method so everything is consistent (can do in a follow-up PR (also made this comment on the PR that introduced this change)